services/docker-compose-ops.yml

```diff
@@ -82,7 +82,8 @@ services:
         user_notifications:${REDIS_HOST}:${REDIS_PORT}:4:${REDIS_PASSWORD},
         announcements:${REDIS_HOST}:${REDIS_PORT}:5:${REDIS_PASSWORD},
         distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6:${REDIS_PASSWORD},
-        deferred_tasks:${REDIS_HOST}:${REDIS_PORT}:7:${REDIS_PASSWORD}
+        deferred_tasks:${REDIS_HOST}:${REDIS_PORT}:7:${REDIS_PASSWORD},
+        dynamic_services:${REDIS_HOST}:${REDIS_PORT}:8:${REDIS_PASSWORD}
```

Comment:
Does it require some changes in OPS?

Reply:
And could you explain to me what is the usage of the new redis database? Our of curiousity